pdvega/plotting.py

```diff
@@ -35,78 +33,80 @@ def scatter_matrix(frame, c=None, s=None, figsize=None, dpi=72.0, **kwds):
     """
     if kwds:
-        warnings.warn("Unrecognized keywords in pdvega.scatter_matrix: {0}"
-                      "".format(list(kwds.keys())))
-    cols = [col for col in frame.columns
-            if col not in [c, s]
-            if infer_vegalite_type(frame[col], ordinal_threshold=0) == 'quantitative']
+        warnings.warn(
+            "Unrecognized keywords in pdvega.scatter_matrix: {0}"
+            "".format(list(kwds.keys()))
+        )
+
+    cols = [
+        col
+        for col in frame.columns
+        if col not in [c, s]
+        if infer_vegalite_type(frame[col], ordinal_threshold=0) == "quantitative"
+    ]
+
     spec = {
-      "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
-      "repeat": {
-        "row": cols,
-        "column": cols[::-1]
-      },
-      "spec": {
-        "mark": "point",
-        "selection": {
-          "brush": {
-            "type": "interval",
-            "resolve": "union",
-            "on": "[mousedown[event.shiftKey], window:mouseup] > window:mousemove!",
-            "translate": "[mousedown[event.shiftKey], window:mouseup] > window:mousemove!",
-            "zoom": "wheel![event.shiftKey]"
-          },
-          "grid": {
-            "type": "interval",
-            "resolve": "global",
-            "bind": "scales",
-            "translate": "[mousedown[!event.shiftKey], window:mouseup] > window:mousemove!",
-            "zoom": "wheel![!event.shiftKey]"
-          }
-        },
-        "encoding": {
-          "x": {"field": {"repeat": "column"}, "type": "quantitative"},
-          "y": {"field": {"repeat": "row"}, "type": "quantitative"},
-          "color": {
-            "condition": {
-              "selection": "brush",
+        "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+        "repeat": {"row": cols, "column": cols[::-1]},
+        "spec": {
+            "mark": "point",
+            "selection": {
+                "brush": {
+                    "type": "interval",
+                    "resolve": "union",
+                    "on": "[mousedown[event.shiftKey], window:mouseup] > window:mousemove!",
+                    "translate": "[mousedown[event.shiftKey], window:mouseup] > window:mousemove!",
+                    "zoom": "wheel![event.shiftKey]",
+                },
+                "grid": {
+                    "type": "interval",
+                    "resolve": "global",
+                    "bind": "scales",
+                    "translate": "[mousedown[!event.shiftKey], window:mouseup] > window:mousemove!",
+                    "zoom": "wheel![!event.shiftKey]",
+                },
+            },
+            "encoding": {
+                "x": {"field": {"repeat": "column"}, "type": "quantitative"},
+                "y": {"field": {"repeat": "row"}, "type": "quantitative"},
+                "color": {"condition": {"selection": "brush"}, "value": "grey"},
```

Comment:
Any reason this isn't being built in Altair?

Reply:
OK, I'm fine leaving this as-is for now.

Reply:
great!  I have this and a couple of other advancements (like symbol mapping for scatterplots, etc)
on my list for the next PR